x-pack/plugins/lens/public/app_plugin/app.tsx

```diff
@@ -425,6 +425,14 @@ export function App({
           isSaveModalVisible: false,
           isLinkedToOriginatingApp: false,
         }));
+        notifications.toasts.addSuccess(
```

Comment:
You put the same notification in this conditional branch and in the main one (https://github.com/elastic/kibana/pull/80788/files#diff-62b4eca6ab93c938023295869419d9b3031fd1c11470d5a3fec89f0012bbf036R451-R458) - it seems like putting just a single invocation in front of the branched code (line 413 of this file) would achieve the same result. Is there any reason I'm missing you didn't do this?

Reply:
You're right! I was actually following where `isSaveModalVisible: false` was to display the toast notification when the save modal is closed. I just tried your suggestion, and it indeed achieves the same result. I also did a manual test and ran `node scripts/jest lens` to verify the suggested change.